src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -180,6 +180,13 @@ public WebSocketsTransport(WebSocketOptions options, Channel<byte[]> application
 
                             await ws.SendAsync(new ArraySegment<byte>(buffer), _options.WebSocketMessageType, endOfMessage: true, cancellationToken: CancellationToken.None);
                         }
+                        catch (WebSocketException socketException)
+                        when (socketException.WebSocketErrorCode == WebSocketError.InvalidState)
```

Comment:
shouldn't you also check that the transport/channel was actually closed to avoid swallowing exceptions in case where they should be thrown?